prisma/.env

```diff
@@ -0,0 +1 @@
+MAIN_DATABASE_URL="mysql://user:password@localhost:3306/db"
```

Comment:
you should add this file to .gitignore
so create one and add this file and push the commits to the pr
don't forget to rm this file from the git stage